src/EditorFeatures/Core/GoToDefinition/GoToDefinitionHelpers.cs

```diff
@@ -127,23 +127,7 @@ internal static class GoToDefinitionHelpers
 
             return threadingContext.JoinableTaskFactory.Run(() =>
                 streamingPresenter.TryNavigateToOrPresentItemsAsync(
-                    threadingContext, solution.Workspace, title, definitions, cancellationToken));
-        }
-
-        public static bool TryGoToDefinition(
-            ImmutableArray<DefinitionItem> definitions,
-            Workspace workspace,
-            string title,
-            IThreadingContext threadingContext,
-            IStreamingFindUsagesPresenter streamingPresenter,
-            CancellationToken cancellationToken)
-        {
-            if (definitions.IsDefaultOrEmpty)
-                return false;
-
-            return threadingContext.JoinableTaskFactory.Run(() =>
-                streamingPresenter.TryNavigateToOrPresentItemsAsync(
-                    threadingContext, workspace, title, definitions, cancellationToken));
+                    threadingContext, solution, title, definitions, cancellationToken));
```

Comment:
this method went away as it was identical to the one above (which already took a solution).